Library/Homebrew/tap_migrations.rb

```diff
@@ -113,6 +116,7 @@
   "opengrm-ngram" => "homebrew/science",
   "pan" => "homebrew/boneyard",
   "pari" => "homebrew/x11",
+  "pathfinder" => "homebrew/boneyard",
```

Comment:
I couldn't find any alternative sources for this; the homepage seems AWOL as well.